SonarQube.TeamBuild.PreProcessor/Roslyn/Model/AdditionalFile.cs

```diff
@@ -0,0 +1,30 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="AdditionalFile.cs" company="SonarSource SA and Microsoft Corporation">
+//   Copyright (c) SonarSource SA and Microsoft Corporation.  All rights reserved.
+//   Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+//-----------------------------------------------------------------------
+
+using System.Xml;
+using System.Xml.Serialization;
+
+namespace SonarQube.TeamBuild.PreProcessor.Roslyn
+{
+    /// <summary>
+    /// XML-serializable data class for a single analyzer AdditionalFile
```

Comment:
It would be more helpful if you could explain what an Aditional File is in the context of an analyser.